options/locale/locale_en-US.ini

```diff
@@ -1038,7 +1038,7 @@ migrated_from_fake = Migrated From %[1]s
 migrate.migrate = Migrate From %s
 migrate.migrating = Migrating from <b>%s</b> ...
 migrate.migrating_failed = Migrating from <b>%s</b> failed.
-migrate.migrating_failed.error = Error: %s
+migrate.migrating_failed.error = Failed to migrate: %s
```

Comment:
Is there a way to tell crowdin they should translate if the key wasn't changed?

Reply:
Crowdin always does so.